lms/djangoapps/verify_student/management/commands/send_verification_email_test.py

```diff
@@ -0,0 +1,78 @@
+"""
+Django admin command to send verification approved email to learners
```

Comment:
@awaisdar001 would it be better if a unit test runs this management command instead of running it manually to test?

Reply:
I am not going to merge this magm command to production, this is just to test the emails on the sandbox.